Dockerfile

```diff
@@ -28,9 +28,11 @@ ENV DEBIAN_FRONTEND=noninteractive
 #   unixodbc: required for odbc driver datasources (i.e. Microsoft SQL Server)
 #   msodbcsql18: specific SQL Server odbc driver
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
-RUN apt-get update && apt-get install --no-install-recommends -y \
+RUN apt-get update && \
+    CURL_VERSION=$(apt-cache madison curl | head -1 | awk '{print $3}') && \
+    apt-get install --no-install-recommends -y \
       python3-dev=3.13.5-1 gcc=4:14.2.0-1 \
-      curl=8.14.1-2 gnupg=2.4.7-21 ca-certificates=20250419 unixodbc=2.3.12-2 \
+      "curl=${CURL_VERSION}" gnupg=2.4.7-21 ca-certificates=20250419 unixodbc=2.3.12-2 \
```

Comment:
I tried using an unpinned version, but got a `hadolint` [warning](https://github.com/great-expectations/cloud/actions/runs/19436252824/job/55607740220), setting a variable to the version of curl available satisfies hadolint

Reply:
Yes please